sdk/recoveryservicessiterecovery/azure-resourcemanager-recoveryservicessiterecovery/src/main/java/com/azure/resourcemanager/recoveryservicessiterecovery/fluent/StorageClassificationMappingsClient.java

```diff
@@ -0,0 +1,337 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.resourcemanager.recoveryservicessiterecovery.fluent;
+
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.http.rest.Response;
+import com.azure.core.management.polling.PollResult;
+import com.azure.core.util.Context;
+import com.azure.core.util.polling.SyncPoller;
+import com.azure.resourcemanager.recoveryservicessiterecovery.fluent.models.StorageClassificationMappingInner;
+import com.azure.resourcemanager.recoveryservicessiterecovery.models.StorageClassificationMappingInput;
+
+/** An instance of this class provides access to all the operations defined in StorageClassificationMappingsClient. */
+public interface StorageClassificationMappingsClient {
```

Comment:
Renamed from `ReplicationStorageClassificationMappingsClient` to `StorageClassificationMappingsClient`.

Reply:
Is it done manually or in codegen?

Reply:
It's through local codegen.

autorest PR: https://github.com/Azure/autorest.java/pull/1911
specs PR: https://github.com/Azure/azure-rest-api-specs/pull/22040